srcpkgs/pmbootstrap/template

```diff
@@ -10,7 +10,8 @@ maintainer="bra1nwave <bra1nwave@protonmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://postmarketos.org"
 distfiles="https://gitlab.com/postmarketOS/pmbootstrap/-/archive/${version}/pmbootstrap-${version}.tar.gz"
-checksum=7288217212dbb89d6623718561440c3593a99c298db631912ea4a192ca397fb2
+checksum=b7737afd92ff2e527775556a3f824aa97b64dbf2012de827a8c3dd4b3b1731ba
+
```

Comment:
This newline shouldn't be here.

Reply:
Ah, yes, must have missed that. It's fixed now.